packages/sampling/test/sampler.test.ts

```diff
@@ -0,0 +1,264 @@
+/*
+Copyright 2023-2024 SolarWinds Worldwide, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { randomBytes } from "node:crypto"
+
+import {
+  createTraceState,
+  diag,
+  ROOT_CONTEXT,
+  type Span,
+  SpanKind,
+  trace,
+  TraceFlags,
+} from "@opentelemetry/api"
+import { SamplingDecision } from "@opentelemetry/sdk-trace-base"
+import { describe, expect, it } from "@solarwinds-apm/test"
+
+import {
+  OboeSampler,
+  type SampleParams,
+  SpanType,
+  spanType,
+} from "../src/sampler.js"
+import { Flags, type LocalSettings, type Settings } from "../src/settings.js"
+
+interface MakeSpan {
+  name?: string
+  traceId?: string
+  id?: string
+
+  remote?: boolean
+  sampled?: boolean
+
+  sw?: boolean | "inverse"
```

Comment:
What is the meaning of `inverse`

Reply:
To test when the traceparent and sw tracestate key have inverse value (ie traceparent 01 but sw 00 and vice versa)

Reply:
I see.